### ISSUE TITLE
Build is failed due to error in `from torch.utils._import_utils import dill_available` in dgl

### ISSUE THREAD

**Issue by SarahAlidoost** [SDCCA/DGL-PTM#101]
The build actions failed due to the error:

```bash
 from torch.utils._import_utils import dill_available
E   ModuleNotFoundError: No module named 'torch.utils._import_utils'
```
see log [here](https://github.com/SDCCA/DGL-PTM/actions/runs/10200265667/job/28219702496). 

This is related to torchdata https://pytorch.org/data/beta/index.html.  and reported in https://github.com/dmlc/dgl/issues/7380 and https://github.com/dmlc/dgl/issues/7387.  Those issues in dgl are fixed, but I am not sure if dgl PyPI packages are also updated. 

As a workaround, we can pin the version of `torchdata` to 0.7.

**Comment by vanlankveldthijs** [comment]
Fixed in #81

**Comment by cy-dream** [comment]
torch==2.1.0
torchdata==0.7.0
